x86/Makefile.common

```diff
@@ -85,7 +85,8 @@ tests-common = $(TEST_DIR)/vmexit.$(exe) $(TEST_DIR)/tsc.$(exe) \
                $(TEST_DIR)/tsx-ctrl.$(exe) \
                $(TEST_DIR)/eventinj.$(exe) \
                $(TEST_DIR)/smap.$(exe) \
-               $(TEST_DIR)/umip.$(exe)
+               $(TEST_DIR)/umip.$(exe) \
+	        $(TEST_DIR)/hello_acrn.$(exe)
```

Comment:
Looks like the indentation is slightly off

Reply:
yes, I will update.

Reply:
Done